tasks/quality.js

```diff
@@ -156,30 +145,19 @@ module.exports = function(grunt) {
       }
     }
 
-    // Process eslint.dir paths from config for template placeholders.
-    var eslintPaths = _.map(eslintTarget, function (item) {
-      return grunt.template.process(item);
-    });
-
-    // Only enable eslint if at least one source file is identified when the
-    // configured paths are expanded.
-    if (grunt.file.expand(eslintPaths).length) {
-      grunt.config('eslint', {
+    grunt.config('eslint', {
+      options: {
+        configFile: eslintConfigFile
+      },
+      validate: eslintTarget,
+      analyze: {
         options: {
-          configFile: eslintConfigFile
+          format: 'checkstyle',
+          outputFile: '<%= config.buildPaths.reports %>/eslint.xml'
         },
-        validate: eslintTarget,
-        analyze: {
-          options: {
-            format: 'checkstyle',
-            outputFile: '<%= config.buildPaths.reports %>/eslint.xml'
-          },
-          src: eslintTargetAnalyze
-        }
-      });
-      validate.push(eslintName + ':validate');
```

Comment:
@grayside It looks like eslint issues are causing the build to abort because the use of `eslintName` was lost, which includes the "force:" prefix when `ignoreError` is set. I'll try to re-introduce this use.